samwise.js

```diff
@@ -3,15 +3,15 @@
  *
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.